resources/assets/js/components/UnclaimedJobCard.tsx

```diff
@@ -4,26 +4,30 @@ import { JobStatus } from "../models/lookupConstants";
 
 export interface UnclaimedJobCardProps {
   title: string;
+  url: string;
```

Comment:
I added a `Link` type in the JobCard PR that should probably be broken out somewhere else (models/types maybe? @tristan-orourke) that we should use everywhere for consistency. We often seem to be missing things like titles in our anchors.

Reply:
Good point that we're missing titles in anchors, but I'm not sure we need to pass all that in through component props. Wouldn't it make sense to define the anchor title and text in a <FormattedMessage> right in the job card?

Reply:
Yep, but either way it would be nice to `import { Link } from '../components/Link'` among the various places where we implement an anchor tag so we're not forgetting things.

Reply:
A Link component would be so similar to an anchor element, I'm not convinced its worth it. We added a title attribute in UnclaimedJobCard, its just an empty string, the component wouldn't prevent that.
I feel like what we really want is a linting rule. I did some searching, but couldn't find any for title attributes in anchor tags. We're already using the common accessibility eslint plugin. Maybe we should write our own rule?

Reply:
I don't think anything will prevent us from deliberately passing empty strings to props, but some sort of component would at least throw a TS error if a required prop (title) is missing. If you can find/write a linting rule for that I'd be onboard.

Reply:
I followed your suggestion @Khristophor and extracted Link into an app-wide interface, and used it in UnclaimedJobCard. Its still easy enough to set the anchor title attribute to an empty string, which we might do here anyway, but we'll have to be deliberate about it.